linux_os/guide/system/auditing/auditd_configure_rules/directory_group_ownership_var_log_audit/oval/shared.xml

```diff
@@ -0,0 +1,46 @@
+<def-group>
+  <definition class="compliance" id="directory_group_ownership_var_log_audit" version="1">
+    {{{ oval_metadata("Checks that all /var/log/audit directories are group owned by the root user.") }}}
+    <criteria operator="OR">
+      <criteria operator="AND" comment="directories are root group owned">
+        <criterion test_ref="test_group_ownership_var_log_audit_directories" />
+      </criteria>
+      <criteria operator="AND" comment="log_group in auditd.conf is not root">
+        <extend_definition comment="log_group in auditd.conf is not root"
+        definition_ref="auditd_conf_log_group_not_root" />
+        <criterion test_ref="test_group_ownership_var_log_audit_directories-non_root" />
```

Comment:
In this scenario https://paste.centos.org/view/7e2744b8 the /var/log/audit is owned by evil people but the rule passes but I think it should fail.

Reply:
In my mind, shouldn't we accept whatever `log_group` is specified in that file and have a separate rule to audit that `log_group` is acceptably either `root` or `adm`... But I dunno :)

Reply:
this way the pastebin is going to expire so I will just copy paste it here:

```
#!/bin/bash
 
groupadd group_test
groupadd evil_people
 
sed -i "/\s*log_group.*/d" /etc/audit/auditd.conf
echo "log_group = group_test" >> /etc/audit/auditd.conf
 
chgrp evil_people /var/log/audit
```

Reply:
This is a copy paste from the previous implementation. I don't think OVAL would be powerful enough to do such check where it would detect exactly the group that is set in the log file.

I don't know how to proceed here... maybe we could remove this criterion since STIG does not describe it.